src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Sami32 CRF? There are many different i7's, and many other options one can tune than the resolution, so I have a hard time to believe that all i7's have problems with all 4K transcoding. I'd also think that GPU acceleration could play a major role here, although the typical UMS computer might not have a fast GPU.